src/FluentMigrator.Runner.Postgres/Processors/Postgres/PostgresProcessor.cs

```diff
@@ -71,6 +71,23 @@ public class PostgresProcessor : GenericProcessorBase
             _quoter = new PostgresQuoter(pgOptions);
         }
 
+        public PostgresProcessor(
```

Comment:
I never found myself needing to add a constructor such as this, could you please explain to me it's purpose?
One thing I did add but I don't see here, was an addition to `PostgtesrunnerBuilderExtensions.cs` such as:
```
        /// <summary>
        /// Adds Postgres 10.0 support
        /// </summary>
        /// <param name="builder">The builder to add the Postgres-specific services to</param>
        /// <returns>The migration runner builder</returns>
        public static IMigrationRunnerBuilder AddPostgres100(this IMigrationRunnerBuilder builder)
        {
            builder.Services
                .AddScoped<Postgres100Processor>()
                .AddScoped<IMigrationProcessor>(sp => sp.GetRequiredService<Postgres100Processor>())
                .AddScoped<Postgres100Generator>()
                .AddScoped<IMigrationGenerator>(sp => sp.GetRequiredService<Postgres100Generator>());
            return builder.AddCommonPostgresServices();
        }
```

How do you configure Postgres10* components to be used?  Do you not use the extension methods like this and other platforms provide?